RELEASE.md

```diff
@@ -0,0 +1,8 @@
+On branch `main`
+- bump version in `package.json`
+- run `npm i` to update `package-lock.json`
+- if it exists, update "Unreleased" header in `CHANGELOG.md` to the new version number
+- commit with a message `:package: <version-number>`. (example `":package: 1.0.2"`)
+- push the commit to `main`
+- run `gh-release` (may need to install)
```

Comment:
Could you clarify which one we need to install? Googling I already found 2: 
- https://cli.github.com/manual/gh_release and https://github.com/cli/cli#installation
- https://github.com/hypermodules/gh-release

Reply:
Good point.